### YAML file for the MQTT configuration

device_ID: '00'
base_ID: '01'

mqtt_Broker: '117.184.79.186'  
mqtt_Port: 18383
mqtt_keepalive: 60 
mqtt_UserName: 'antobot'
mqtt_PassWord: '1qaz@WSX'





